src/test/codegen/repr-u128.rs

```diff
@@ -0,0 +1,20 @@
+// ignore-windows
+//min-system-llvm-version 8.0
+
+//compile-flags: -g -C no-prepopulate-passes
+#![feature(repr128)]
+
+#[repr(u128)]
+pub enum Foo {
+    Lo,
+    Hi = 1 << 64,
+    Bar = 18_446_745_000_000_000_123,
+}
+
+// CHECK: {{.*}}DIDerivedType{{.*}}tag: DW_TAG_member,{{.*}}
+// name: "None",{{.*}}extraData:18446745000000000124
```

Comment:
Does this work split on two lines like this?
Feel free to add an `// ignore-tidy-linelength` comment at the top of the file if the line is too long.

Reply:
I added it, thanks!